configure

```diff
@@ -709,6 +709,7 @@ OBJECTS = $(patsubst src/%.cpp,build/%.o,$(SOURCES))
 EXAMPLES := examples/Single_energy/single_energy \
             examples/Multiple_energy/multiple_energy \
             examples/Atm_default/atm_default \
+	    examples/Emitting_Atmosphere/emitting_atmosphere \
```

Comment:
It looks like this line has gotten a tab character in it, when it should be all spaces for alignment.